opendatahub-io/odh-dashboard
# Create Run | Create Experiment | View Pipeline Run

Closes:
* https://github.com/opendatahub-io/odh-dashboard/issues/1183
* https://github.com/opendatahub-io/odh-dashboard/issues/1064
* https://github.com/opendatahub-io/odh-dashboard/issues/1116 (edit / delete missing)
* https://github.com/opendatahub-io/odh-dashboard/issues/1118 (selected node details missing)

## Description


Lots covered here.

![Screenshot 2023-05-01 at 1 52 57 PM](https://user-images.githubusercontent.com/8126518/235505411-5dadc414-4405-4493-92c4-e2b9f95a8b46.png)
![Screenshot 2023-05-01 at 1 53 01 PM](https://user-images.githubusercontent.com/8126518/235505412-43569075-27c2-4ca6-aac4-e4a3f58d272f.png)
![Screenshot 2023-05-01 at 1 53 09 PM](https://user-images.githubusercontent.com/8126518/235505413-d456a93b-0bcd-44e7-9cb2-222a759783ce.png)
![Screenshot 2023-05-01 at 2 18 21 PM](https://user-images.githubusercontent.com/8126518/235505414-7b240cf1-11e3-4a6d-81cf-63d5876f46ec.png)
![Screenshot 2023-05-01 at 2 18 27 PM](https://user-images.githubusercontent.com/8126518/235505417-969e683f-e3cc-4f4e-bcce-219469c9426b.png)
![Screenshot 2023-05-01 at 2 18 32 PM](https://user-images.githubusercontent.com/8126518/235505418-33f97ab2-b659-4052-a191-8ba04c2562e5.png)
![Screenshot 2023-05-01 at 2 19 26 PM](https://user-images.githubusercontent.com/8126518/235505419-e029b850-80df-4239-b6fe-dabeec0400b0.png)
![Screenshot 2023-05-01 at 2 19 57 PM](https://user-images.githubusercontent.com/8126518/235505420-7c76fc3f-3e3d-4428-917b-f5d257b5aed2.png)
![Screenshot 2023-05-01 at 2 20 11 PM](https://user-images.githubusercontent.com/8126518/235505421-2bc65efd-58ee-46d0-9232-ebad3df2cfbc.png)
![Screenshot 2023-05-01 at 2 20 22 PM](https://user-images.githubusercontent.com/8126518/235505423-95420e4c-339c-48b6-97a7-ddc6eb6d7ec8.png)
![Screenshot 2023-05-01 at 2 21 42 PM](https://user-images.githubusercontent.com/8126518/235505491-8e405f28-2d3c-47f7-a54c-afc912de277d.png)


## How Has This Been Tested?



* View Runs => Find a run and click on it
* Create Runs
    * => Global Runs page => Create Run
    * => View Runs => Kebab => Clone Run
* Create Experiments
    * => Global Runs page => Kebab => Create Experiment
    * => Create Run page
* See https://github.com/opendatahub-io/odh-dashboard/issues/1183 for details on the improvements

## Test Impact


Testing is handled after

## Request review criteria:




/cc @Gkrumbach07
